hs_app_netCDF/models.py

```diff
@@ -341,6 +341,15 @@ def get_hs_term_dict(self):
                 break
         return hs_term_dict
 
+    # return single file name included in the resource
+    def get_res_file_name(self):
+        for res_file in self.files.all():
+            _, f_fullname, f_ext = get_resource_file_name_and_extension(res_file)
+            if f_ext.lower() == '.nc':
+                return f_fullname
+
+        return ''
```

Comment:
If the code gets here (no file in the the resource with .nc extension), is the resource invalid?  The way this is coded, it feels like there is a strong dependency on this file being in the resource.  But the way it works is everything will still go through without any awareness or notification relayed to the user.